### PR TITLE
build: Rename Atala Prism to Hyperledger Identus

### PR DESCRIPTION
**Pull request by FabioPinheiro**
### Description: 

Rename Atala Prism to Hyperledger Identus

Also This is a bit of a breaking change for deployments

TODO Add commit to .git-blame-ignore-revs but I need to do merge commit...

### Checklist: 
- [x] My PR follows the contribution guidelines of this project
- [x] My PR is free of third-party dependencies that don't comply with the [Allowlist](https://toc.hyperledger.org/governing-documents/allowed-third-party-license-policy.html#approved-licenses-for-allowlist)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked the PR title to follow the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/)

### CODE REVIEW

**Review by petevielhaber — APPROVED** [review]

File: webapp/src/main/scala/io/iohk/atala/mediator/App.scala [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package io.iohk.atala.mediator
```
Comment (mineme0110) [inline webapp/src/main/scala/io/iohk/atala/mediator/App.scala:1]: https://github.com/input-output-hk/atala-prism-mediator/tree/rename_to_identus/webapp 
the folder ins still atala
webapp/src/main/scala/io/iohk/atala/mediator/App.scala

File: docker-compose.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ services:
       - ./initdb.js:/docker-entrypoint-initdb.d/initdb.js
 
   atalaprism-mediator:
```
Comment (mineme0110) [inline docker-compose.yml]: identus

File: docs/architecture/structurizr/workspace.dsl [hunk 3]
```diff
@@ -4,7 +4,7 @@ workspace {
         agent = softwareSystem "Cloud / Edge Agent" "" "External"
 
         enterprise "Atala" {
```
Comment (mineme0110) [inline docs/architecture/structurizr/workspace.dsl]: Identus ?

File: webapp/src/main/scala/io/iohk/atala/mediator/AppUtils.scala [hunk 4]
```diff
@@ -83,11 +84,12 @@ object AppUtils {
                 // src := "https://atalaprism.io/images/atala-prism-logo-suite.svg",
```
Comment (mineme0110) [inline webapp/src/main/scala/io/iohk/atala/mediator/AppUtils.scala]: may be remove this or fix it identus

Comment (FabioPinheiro) [reply webapp/src/main/scala/io/iohk/atala/mediator/AppUtils.scala]: I wanted to fix but there is no Identus logo ATM

File: webapp/src/main/scala/io/iohk/atala/mediator/App.scala (hunk 1 above)
Comment (FabioPinheiro) [reply webapp/src/main/scala/io/iohk/atala/mediator/App.scala:1]: Thx I updated this

**Review by mineme0110 — APPROVED** [review]

**Pull request merged**